seagoat/repository.py

```diff
@@ -98,7 +103,7 @@ def analyze_files(self):
                     if not is_file_type_supported(filename):
                         continue
 
-                    if not (self.path / filename).exists():
+                    if filename not in git_files:
```

Comment:
@kantord maybe we also should skip files excluded by ignorePatterns right here?

Reply:
yes, but then we should also remove this logic:

```
    def _is_file_ignored(self, path: str):
        for pattern in self.config["server"]["ignorePatterns"]:
            if Path(path).match(pattern):
                return True

        return False
```